src/Popper.js

```diff
@@ -1,21 +1,25 @@
 // @flow
 import deepEqual from "deep-equal";
 import * as React from 'react';
-import PopperJS, {
+import { createPopper } from '@popperjs/core';
+import {
+  type State,
   type Placement,
+  type PositioningStrategy,
   type Instance,
-  type Data,
-  type Modifiers,
-  type ReferenceObject,
-} from 'popper.js';
+  type VirtualElement,
+  type Modifier,
+  type ModifierArguments,
+} from '@popperjs/core/lib';
```

Comment:
I think importing from `@popperjs/core` should be enough

Reply:
unfortunately it doesn't work, popper exports from main file `@popperjs/core` only three things: 
```js
export { createPopper, popperGenerator, defaultModifiers };
```

Reply:
See [this one](https://github.com/popperjs/popper-core/blob/49d2ee9bb6d58f43ca1d5183095a1af253fe9aa7/src/popper.js#L28) + [official doc](https://popper.js.org/docs/v2/typings/)